test/functional/api/v2/template_controller_test.rb

```diff
@@ -1,8 +1,30 @@
 require 'test_plugin_helper'
+require 'git'
 
 module Api
   module V2
     class TemplateControllerTest < ::ActionController::TestCase
+
+      private
+      def create_local_template_in_core(t_filename:, t_name:, model: "ProvisioningTemplate", kind: "provision", org: "", loc: "")
```

Comment:
It has no impact on tests, but it is customary to put private methods at the end, because the `private` makes all methods below it private:

```ruby
2.3.4 :010 > class Cow
2.3.4 :011?>   private
2.3.4 :012?>   def crazy_moo
2.3.4 :013?>     "mooooah"
2.3.4 :014?>     end
2.3.4 :015?>   def moo
2.3.4 :016?>     "moo"
2.3.4 :017?>    end
2.3.4 :018?>  end
 => :moo 
2.3.4 :019 > c = Cow.new
 => #<Cow:0x000000025602c8> 
2.3.4 :020 > c.crazy_moo
NoMethodError: private method `crazy_moo' called for #<Cow:0x000000025602c8>
	from (irb):20
	from /home/vagrant/.rvm/rubies/ruby-2.3.4/bin/irb:11:in `<main>'
2.3.4 :021 > c.moo
NoMethodError: private method `moo' called for #<Cow:0x000000025602c8>
	from (irb):21
	from /home/vagrant/.rvm/rubies/ruby-2.3.4/bin/irb:11:in `<main>'
2.3.4 :022 >
```

You could use `public` to make methods public again:

```ruby
2.3.4 :001 > class Cow
2.3.4 :002?>   private
2.3.4 :003?>   def crazy_moo
2.3.4 :004?>     "moooah"
2.3.4 :005?>     end
2.3.4 :006?>   public
2.3.4 :007?>   def moo
2.3.4 :008?>     "mooo"
2.3.4 :009?>    end
2.3.4 :010?>  end
 => :moo 
2.3.4 :011 > c = Cow.new
 => #<Cow:0x0000000147c358> 
2.3.4 :012 > c.crazy_moo
NoMethodError: private method `crazy_moo' called for #<Cow:0x0000000147c358>
	from (irb):12
	from /home/vagrant/.rvm/rubies/ruby-2.3.4/bin/irb:11:in `<main>'
2.3.4 :013 > c.moo
 => "mooo"
```